filebeat/module/apache/access/_meta/fields.yml

```diff
@@ -0,0 +1,27 @@
+- name: access
+  type: group
+  description: >
+    Contains fields for the Apache HTTP Server access logs.
+  fields:
+    - name: remote_ip
```

Comment:
@webmat We should follow up on this and convert it to ECS.

Reply:
Yes, I have a note at the end of the Fb modules list to convert to using `.address` in all remaining places (it was a beta 2 addition, and came from the first migrations, like this one)